dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_postgresql.sql

```diff
@@ -379,6 +379,8 @@ CREATE TABLE t_ds_process_definition_log (
   PRIMARY KEY (id)
 ) ;
 
+create UNIQUE index idx_process_definition_log_code_version on t_ds_process_definition_log (code,version);
```

Comment:
Should we keep the same name as other database type?
```suggestion
create UNIQUE index uniq_idx_code_version on t_ds_process_definition_log (code,version);
```